extensions/community_extensions/skadnetwork.md

```diff
@@ -0,0 +1,850 @@
+# SKAdNetwork
+
+Sponsors: MoPub, Fyber, Unity
+
+## Overview
+
+The IAB Tech Lab has introduced technical specifications aimed at adapting Apple’s [SKAdNetwork][1], a method for validating advertiser app installations, for programmatic ad buying. The OpenRTB object extensions, APIs and file formats described in this document together enable the advertising ecosystem to communicate and manage the information needed to use the SKAdNetwork capabilities in iOS 14 and above.
+
+The following are the updates provided in this document
+1. A Bid Request extension (`BidRequest.imp.ext.skadn`)
```

Comment:
If we cannot move this to the `app` object, may be worth noting for clarity that, if the object appears in several `imp` objects, the contents will be exactly the same.

Reply:
That's a good callout to discuss tomorrow. Thanks for the point on that!